gulpfile.js

```diff
@@ -0,0 +1,77 @@
+var gulp = require('gulp');
+var sass = require('gulp-sass'); //compiles Sass
+var browserSync = require('browser-sync').create(); //listens for changes and refreshes the brwoser
+var useref = require('gulp-useref'); //puts all JS scripts into 1 JS script
+var uglify = require('gulp-uglify'); // minifies JS files
+var gulpIf = require('gulp-if') // checks if a files is a particular type for minification
+var cssnano = require('gulp-cssnano') // minifies CSS
+var del = require('del'); // deletes files/folders as you request
+var runSequence = require('run-sequence'); // allows for you to order the process
+var babel = require('gulp-babel'); // allows you to convert old JS to ES6
+var sourcemaps = require('gulp-sourcemaps'); // allows you to use sourcempas to debugg
+var autoprefixer = require('gulp-autoprefixer'); // adds prefixes to CSS files
+var jshint = require('gulp-jshint'); // lints JS files
+
+gulp.task('sass', function() {
+    return gulp.src('app/scss/**/*.scss')
+        .pipe(sourcemaps.init())
+        .pipe(sass())
+        .pipe(sourcemaps.write('../maps'))
+        .pipe(autoprefixer())
+        .pipe(gulp.dest('app/css'))
+        .pipe(browserSync.reload({
+            stream: true
+    }))
+})
+
+gulp.task('browserSync', function() {
+    browserSync.init({
+        server: {
+            baseDir: "app"
+        }
+    });
+});
+
+gulp.task('useref', function() {
+    return gulp.src('app/*.html')
+        .pipe(useref())
+        .pipe(gulpIf('*.js', jshint()))
+        .pipe(gulpIf('*.js', jshint.reporter('default')))
+        .pipe(gulpIf('*.js', uglify()))
+        .pipe(gulpIf('*.css', cssnano()))
+        .pipe(gulp.dest('dist'))
```

Comment:
I am not sure about this. I am trying to think through the implications of having a separate `dist` directory for the deployment on commerce.gov. We would either have to get them to install Node.js (unlikely) and run the build step, or check in the `dist` directory.

It is not ideal, but the simplest solution for deployment might be to have the web pages permanently use the minified assets. It makes debugging a little harder, but it can be worked around.

cc @noonkay